boms/cloud-oss-bom/RELEASING.md

```diff
@@ -77,7 +77,9 @@ run this command from a g4 client to retry the Rapid build without going all the
 back to the beginning:
 
 ```
-$ blaze run java/com/google/cloud/java/tools:ReleaseBom -- --version=${VERSION}
+$ blaze run java/com/google/cloud/java/tools:ReleaseRapidProject -- \
+    --project_name=cloud-java-tools-cloud-opensource-java-bom-kokoro-release \
+    --version=${VERSION} --committish_suffix=bom
```

Comment:
"committish_suffix" feels wrong as if it's expsing implementation details in the UI. This should probably be either suffix or more likely the whole commitish.

Reply:
Updated for committish. Removed version flag.